WordPress/WordPressShare/UIImageView+Extensions.swift

```diff
@@ -0,0 +1,122 @@
+import Foundation
+
+
+extension UIImageView
+{
+    /// Downloads an image and updates the UIImageView Instance
+    ///
+    /// - Parameters:
+    ///     - url: The URL of the target image
+    ///     - placeholderImage: Image to be displayed, temporarily, while the Download OP is executed
+    ///
+    public func downloadImage(url: NSURL?, placeholderImage: UIImage?) {
```

Comment:
You're right. The reason why it was an optional, it was merely convenience. But i really like the way your suggestion looks like. Updating, thank you!